src/actions-on-google.ts

```diff
@@ -675,8 +686,15 @@ export class ActionsOnGoogle {
                         }
                     }
                 }
+                if (data.screen_out && data.screen_out.format === 'HTML' && data.screen_out.data) {
+                    assistResponse.screenOutHtml = data.screen_out.data.toString()
+                }
             })
             conversation.on('end', () => {
+                if (audioBuffers.length > 0) {
+                    assistResponse.audioOut = Buffer.concat(audioBuffers)
```

Comment:
Is there a reason to add an `audioOut` buffer to the response? It was omitted for the primary reason that it wouldn't be particularly helpful.

Reply:
Same reason as above - listening to the audio response will help users in composing test cases.

Reply:
Will make this an option.